### PR TITLE
fix(Core/Spells): Fixed players being able to mount with all transfor…

### LINKED ISSUE #10395: I can't get on a horse when I use ShadowDance
### What client do you play on?
Original Post: https://github.com/chromiecraft/chromiecraft/issues/2883
ruRU

### Faction

- [X] Alliance
- [X] Horde

### Content Phase:

- [ ] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59
- [X] 60

### Current Behaviour

I can't get on a horse when I use ShadowDance (last talent of Rogue).

### Expected Blizzlike Behaviour

you can ride a horse in Shadowdance

### Source

_No response_

### Steps to reproduce the problem

1. Use Shadowdanse
2. Try to use mount

### Extra Notes

_No response_

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/f3774fd9ac8241994396426e6c5f9246ce56400b

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [mod-save-inventory](https://github.com/Nyeriah/mod-save-inventory)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)
- [lua-zone-debuff](https://github.com/55Honey/Acore_ZoneDebuff)

### Customizations

None

### Server

ChromieCraft

### PR DESCRIPTION
**Pull request by UltraNix**
…m auras. Original author: @Shauren

Fixes #10395

## Issues Addressed:

- Closes #10395
- Closes https://github.com/chromiecraft/chromiecraft/issues/2883

## Tests Performed:

- Tested ingame.

## How to Test the Changes:

Use Shadowdanse
Try to use mount

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Shauren** [comment]
Please prefer to use --author argument to `git commit` (for example `git commit --author='Shauren <shauren.trinity@gmail.com>'` ) instead @mentioning me using github username

**Comment by UltraNix** [comment]
@Shauren Done.

File: data/sql/updates/pending_db_world/rev_1652621127582675200.sql [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+`NPCItemDisplay6` INT UNSIGNED NOT NULL DEFAULT '0',
+`NPCItemDisplay7` INT UNSIGNED NOT NULL DEFAULT '0',
+`NPCItemDisplay8` INT UNSIGNED NOT NULL DEFAULT '0',
+`NPCItemDisplay9` INT UNSIGNED NOT NULL DEFAULT '0',
+`NPCItemDisplay10` INT UNSIGNED NOT NULL DEFAULT '0',
+`NPCItemDisplay11` INT UNSIGNED NOT NULL DEFAULT '0',
+`Flags` INT UNSIGNED NOT NULL DEFAULT '0',
+`BakeName` VARCHAR(100) NOT NULL,
+PRIMARY KEY (`ID`) )
+ENGINE=MYISAM CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
```
Comment (pklloveyou) [inline data/sql/updates/pending_db_world/rev_1652621127582675200.sql:26]: CIERROR 1273 (HY000) at line 769: Unknown collation: 'utf8mb4_0900_ai_ci'

**Comment by Nyeriah** [comment]
Must have something to do with old dependencies being used by that OS. Not sure we can do anything about it, all of the other builds are passing.

**Comment by mpfans** [comment]
I suggest merging first, now Ubuntu 18 04 has been abandoned

**Review by Kitzunu — APPROVED** [review]

**Comment by pklloveyou** [comment]
Tested

**Pull request merged**